IFC4x3/Sections/Domain specific data schemas/Schemas/IfcPortsAndWaterways/PropertySets/Pset_CargoCommon/DocPropertySet.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CargoCommon" UniqueId="cccbde5d-aea2-4e5d-a55a-7602871d2687" ApplicableType="IfcTransportElement,IfcTransportElementType" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CargoCommon" UniqueId="cccbde5d-aea2-4e5d-a55a-7602871d2687" ApplicableType="IfcTransportElement/IfcTransportElementNonFixedTypeEnum(.CARGO.),IfcTransportElementType/IfcTransportElementNonFixedTypeEnum(.CARGO.)" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
No problem, if we need to use the normal notation thats fine.
@jmirtsch do you see any issue with not having the enum typing in the ifcxml and psd xmls. When is going to be needed in the encoding??

Reply:
The one thing I would like to resolve here is what to do if we have a select of enums. In that case it makes sense to use the complete type.

There are some queries that rely on this notation in order to function properly. Those will have to be adjusted to incorporate the enum type. From my point of view, it would be sufficient to use FunkyTypeEnum(FUNKY) in those cases but we can go for the dots to nominate a constant as it is in STEP. I would like to restrict this to types where it is actually required e.g. if there are any psets on say IfcFacilityPart or IfcImpactProtectionDevice etc.

I will not be changing the documentation on this yet (time issues) but I would start to look at updating the queries.

Reply:
That sound fine @SergejMuhic, I only applied this to entities with selects, because in IfcDoc the PDT dropdown doesn't work when its a select so required manual updates anyway

Reply:
It works now. :)
![image](https://user-images.githubusercontent.com/16683597/110089513-7f586d00-7d96-11eb-9561-1ab40f0ef207.png)

Will work on the rest of the changes necessary. For now I just adopted your proposed SPF compatible string.

Reply:
Seems I got it covered. Will be looking to commit these changes to the ifcdoc repo over the weekend. Would then probably make sense to release a version (have a couple more changes related to IFC4x3 already).

Reply:
awesome, when its pushed to the Development branch i will update my build of IfcDoc